### PR TITLE
WIP: Bug 1936450: OCP 4.7: Passing oVirt certificate information to Terraform

### PR DESCRIPTION
**Pull request by janoszen**
This PR fixes BZ 1936450. For details please see that BZ.

### CODE REVIEW

File: pkg/tfvars/ovirt/ovirt.go [hunk 1]
```diff
@@ -10,12 +10,14 @@ import (
 	"github.com/openshift/installer/pkg/tfvars/internal/cache"
 )
 
-// Auth is the collection of credentials that will be used by terrform.
+// Auth is the collection of credentials that will be used by terraform.
```
Comment (michalskrivanek) [inline pkg/tfvars/ovirt/ovirt.go:13]: thank you for noticing!:)

File: data/data/ovirt/variables-ovirt.tf [hunk 2]
```diff
@@ -19,6 +19,21 @@ variable "ovirt_password" {
   description = "The plain password of user to access Engine API"
 }
 
+variable "ovirt_cafile" {
```
Comment (staebler) [inline data/data/ovirt/variables-ovirt.tf:22]: `ovirt_cafile` and `ovirt_ca_bundle` need a default since the fields in the go struct are `omitempty`.

**Comment by staebler** [comment]
Please wait for the PR for master to merge and then do an automatic cherry-pick down the releases from there, rather than creating all of the PRs for all of the releases from the start.

**Comment by janoszen** [comment]
> Please wait for the PR for master to merge and then do an automatic cherry-pick down the releases from there, rather than creating all of the PRs for all of the releases from the start.

Thanks, I'll do that.

**Comment by janoszen** [comment]
This needs to be a cherry-pick and will be done when the TF provider is backported.

**Pull request closed without merging**